messaging/tests/test_pub_sub_master.py

```diff
@@ -20,7 +21,7 @@ def test_init(self):
     sm = messaging.SubMaster(events)
     for p in [sm.updated, sm.rcv_time, sm.rcv_frame, sm.alive,
               sm.sock, sm.freq, sm.data, sm.logMonoTime, sm.valid]:
-      self.assertEqual(len(p), len(events))
+      self.assertEqual(len(cast(Sized, p)), len(events))
```

Comment:
all these variables are sized, not sure what's going on here. if you type any one to `dict` in SubMaster it doesn't complain, but if you type them properly (`DIct[str, int]`) then it does. didn't want to go with the first as I'm not sure how it works